src/components/header/HeaderFromLecture.js

```diff
@@ -1,31 +1,30 @@
 import React from 'react';
 import { links, user } from '../../constants';
 import Logo from '../../assets/react.png';
-
 import { UserInfo } from '../user-info/UserInfoFromLecture';
-
 import './Header.scss';
 
+const Links = () => {
+    return links.map(({url,name}, index) => {
+        return (
+            <div key={index} className={"nav-item"}>
+                <a href={url} className={"may-header-links-wrapper-link nav-link"}>{name}</a>
```

Comment:
You can skip using curly brackets around string-values
so it would be just `className="may-header-links-wrapper-link nav-link"`